internal/bgp/frr/config.go

```diff
@@ -345,13 +352,23 @@ func debouncer(body func(config *frrConfig),
 				if !ok { // the channel was closed
 					return
 				}
-				config = newCfg
+				if newCfg.useOld && config == nil {
+					continue // just ignore the event
+				}
```

Comment:
not sure I understand this :/ why sending something to the chan in the first place only to do nothing with it?

Reply:
This should never happen, because it means that we received a "reload old config" event but the old config is nil (meaning, we already tried a reload that failed).
Nonethless, I'd prefer not having the speaker crash but ignore the event in such corner case.

Reply:
sorry, I misread it as `newCfg.config == nil` :sweat_smile: